src/Organisms/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import PriSecBtn from '../../Atoms/PriSecBtn/PriSecBtn';
+import CardWrapper from '../../Atoms/CardWrapper/CardWrapper';
+import RegistrationFormStyled from './RegistrationFormStyled';
+import InputField from '../../Molecules/InputField/InputField';
+import CheckboxAgreeField from '../../Atoms/CheckboxAgreeField/CheckboxAgreeField';
+
+const RegistrationPage = () => {
+  const [name, setName] = useState('');
+  const [surname, setSurname] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [street, setStreet] = useState('');
+  const [zipcode, setZipcode] = useState('');
+  const [city, setCity] = useState('');
```

Comment:
You could set initail state as:
`  const initialState = {
    name: '',
    surname: '',
    email: '',
    password: '',
    passwordRepeat: '',
    street:",
    zipcode:",
    city:"
  };`
  And if there would be anything else you need, and handler could use basic attributes to deal with it (see other comment)